plugin/src/test/resources/SingleBehaviorYes.scala

```diff
@@ -0,0 +1,11 @@
+package org.random.project
+
+import akka.actor.typed.Behavior
+import org.virtuslab.akkasaferserializer.SerializerTrait
+
+object SingleBehaviorYes {
+  @SerializerTrait
```

Comment:
Hmmm actually for some reason, I'm getting `does not extend annotated trait` for all classes that are type params of Behavior, even after annotating `CborSerializable` as `@SerializerTrait` :/

Maybe it's the problem that I needed to add the plugin as both regular dependency (`"org.virtuslab" %% "safer-serializer-plugin" % "0.1.0-SNAPSHOT"`) and compiler plugin (`compilerPlugin("org.virtuslab" %% "safer-serializer-plugin" % "0.1.0-SNAPSHOT")`)? Then maybe `SerializerTrait` referred to from `StdPhase` is not recognized as the same thing as `SerializerTrait` present on the project classpath?...

Reply:
I just did that in my test project and it worked. Maybe the problem was with faulty BFS. I have already replaced it with something more efficient. Could you check again with newest version?
```
addCompilerPlugin("org.virtuslab" %% "safer-serializer-plugin" % "0.1.0-SNAPSHOT")
libraryDependencies += "org.virtuslab" %% "safer-serializer-plugin" % "0.1.0-SNAPSHOT"
```

Reply:
Ok, I take that back. It doesn't work when there are multiple source files.

Reply:
> Ok, I take that back. It doesn't work when there are multiple source files.

Fixed

Reply:
Hmm I've now added a plugin built locally from `fa2c5be80bd81b6880d71fdef831cad156a85aea` to the project (Scala 2.12):

```
    "org.virtuslab" %% "safer-serializer-plugin" % "0.1.0-SNAPSHOT",
    compilerPlugin("org.virtuslab" %% "safer-serializer-plugin" % "0.1.0-SNAPSHOT")
```

I'm getting:


```
[error] ## Exception when compiling 297 sources to /home/plipski/.../target/scala-2.12/classes
[error] java.lang.NoSuchMethodError: 'scala.tools.nsc.reporters.FilteringReporter scala.tools.nsc.Global.reporter()'
[error] org.virtuslab.akkasaferserializer.CheckTypesPluginComponent$$anon$1.apply(CheckTypesPluginComponent.scala:22)
[error] scala.tools.nsc.Global$GlobalPhase.applyPhase(Global.scala:454)
[error] scala.tools.nsc.Global$GlobalPhase.run(Global.scala:402)
[error] scala.tools.nsc.Global$Run.compileUnitsInternal(Global.scala:1511)
[error] scala.tools.nsc.Global$Run.compileUnits(Global.scala:1495)
[error] scala.tools.nsc.Global$Run.compileSources(Global.scala:1488)
[error] scala.tools.nsc.Global$Run.compileFiles(Global.scala:1596)
[error] xsbt.CachedCompiler0.run(CompilerBridge.scala:163)
[error] xsbt.CachedCompiler0.run(CompilerBridge.scala:134)
[error] xsbt.CompilerBridge.run(CompilerBridge.scala:39)
[error] sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:92)
[error] sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$7(MixedAnalyzingCompiler.scala:186)
[error] scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:241)
[error] sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4(MixedAnalyzingCompiler.scala:176)
[error] sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$4$adapted(MixedAnalyzingCompiler.scala:157)
[error] sbt.internal.inc.JarUtils$.withPreviousJar(JarUtils.scala:239)
[error] sbt.internal.inc.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:157)
[error] sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:204)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1(IncrementalCompilerImpl.scala:573)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1$adapted(IncrementalCompilerImpl.scala:573)
[error] sbt.internal.inc.Incremental$.$anonfun$apply$5(Incremental.scala:173)
[error] sbt.internal.inc.Incremental$.$anonfun$apply$5$adapted(Incremental.scala:171)
[error] sbt.internal.inc.Incremental$$anon$2.run(Incremental.scala:458)
[error] sbt.internal.inc.IncrementalCommon$CycleState.next(IncrementalCommon.scala:116)
[error] sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:56)
[error] sbt.internal.inc.IncrementalCommon$$anon$1.next(IncrementalCommon.scala:52)
[error] sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:261)
[error] sbt.internal.inc.Incremental$.$anonfun$incrementalCompile$8(Incremental.scala:413)
[error] sbt.internal.inc.Incremental$.withClassfileManager(Incremental.scala:498)
[error] sbt.internal.inc.Incremental$.incrementalCompile(Incremental.scala:400)
[error] sbt.internal.inc.Incremental$.apply(Incremental.scala:165)
[error] sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:573)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:491)
[error] sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:332)
[error] sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:420)
[error] sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:137)
[error] sbt.Defaults$.compileIncrementalTaskImpl(Defaults.scala:2176)
[error] sbt.Defaults$.$anonfun$compileIncrementalTask$2(Defaults.scala:2133)
[error] sbt.internal.io.Retry$.apply(Retry.scala:40)
[error] sbt.internal.io.Retry$.apply(Retry.scala:23)
[error] sbt.internal.server.BspCompileTask$.compute(BspCompileTask.scala:31)
[error] sbt.Defaults$.$anonfun$compileIncrementalTask$1(Defaults.scala:2129)
[error] scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] sbt.Execute.work(Execute.scala:291)
[error] sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] java.base/java.lang.Thread.run(Thread.java:834)
[error]            
[error] stack trace is suppressed; run last ... / Compile / compileIncremental for the full output
[error] (.... / Compile / compileIncremental) java.lang.NoSuchMethodError: 'scala.tools.nsc.reporters.FilteringReporter scala.tools.nsc.Global.reporter()'
[error] Total time: 115 s (01:55), completed 6 May 2021, 12:32:43
```

Reply:
This is very bad, can't reproduce. May I ask sbt version?

Reply:
sbt is `1.4.4`. I can later provide additional details if needed

Reply:
> Hmm I've now added a plugin built locally from `fa2c5be80bd81b6880d71fdef831cad156a85aea` to the project 

@PawelLipski Fixed by fc5430c5ccf42b965acc2383b2d5f0e03d51715e